packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/Errors/ErrorOverlayLayout/ErrorOverlayLayout.test.tsx

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
Can we change the new files to kebab cases to avoid case insensitive issues in git?